assets/src/application/ui/input/DateTimeRangePicker/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useCallback, useState } from 'react';
+import classNames from 'classnames';
+import { __ } from '@wordpress/i18n';
+
+import { DateTimeRangePicker as DateTimeRangePickerAdapter, DateTimeRangePickerProps } from '@infraUI/inputs';
+import { IconButton, ButtonType } from '../Button';
+import { Save } from '../../display/icons';
+
+const DateTimeRangePicker: React.FC<DateTimeRangePickerProps> = ({
+	className,
+	startDate,
+	endDate,
+	onChange,
+	onChangeValue,
+	...props
+}) => {
+	const [dates, setDates] = useState([startDate, endDate]);
+
+	const onSave: VoidFunction = useCallback(() => {
+		if (typeof onChangeValue === 'function') {
+			onChangeValue(dates);
+		}
+
+		if (typeof onChange === 'function') {
+			onChange(dates);
+		}
+	}, [dates, onChange, onChangeValue]);
+
+	const htmlClass = classNames(
```

Comment:
this is not consistent with the naming we had agreed upon :)

Reply:
this is the same issue as above:

```js
		<div className={htmlClass}>
			<DateTimeRangePickerAdapter required onChange={setDates} startDate={start} endDate={end} {...props} />
```